app/views/outcomes/show.html.erb

```diff
@@ -67,6 +68,7 @@
         <td><%= assessment.minimum_category%></td>
         <td><%= assessment.target_percentage%></td>
         <td><%= assessment.actual_percentage%></td>
+        <td><%= assessment_type(assessment) %></td>
```

Comment:
This would be a bit more round-about, but what do think of `assessment.class.model_name.human`. Then you can control the string with i18n:

```yml
en:
  activerecord:
    models:
      other_assessment: "Other"
      participation: "Program Participation"
```

Definitely more indirection, but that would provide consistent naming in any other place (like error message). The helper is more direct though. Your call.